src/components/ui/Button/Button.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import PropTypes from "prop-types";
+import './Button.css'
+
+const  TFButton = (props) => {
+    const {icon, label, disabled, variant, handleClick, size, customStyles} = props;
+    const mainStyle={};
+    const textStyle = {};
+    const buttonClassName = variant==='secondary' ? disabled? 'secondary-btn-disabled' : 'secondary-btn': disabled ? 'primary-btn-disabled': 'primary-btn';
+    const buttonClassText = variant==='secondary'?(disabled?`secondary-btn-disabled-text`:`secondary-btn-text`):(disabled?`primary-btn-disabled-text`:`primary-btn-text`);
+    const isObjectEmpty = (obj) => {
+      return Object.keys(obj).length === 0;
+    }
+    if(icon!==""){
+      if(isObjectEmpty(customStyles)){
```

Comment:
we dont need this if statement it can happen in the styles object only no need to check if it is empty or not